lib/schema/schema.js

```diff
@@ -253,6 +253,13 @@ exports.convertMgToMmol = function(mg) {
   return mg / 18.01559;
 };
 
+exports.convertMgToMmolPrecision = function(mg) {
+  const mmolLToMgdLConversionFactor = 18.01559;
+  const mmolLToMgdLPrecisionFactor = 100000.0;
+  let mmolVal = Math.round(mg / mmolLToMgdLConversionFactor * mmolLToMgdLPrecisionFactor + 0.5);
```

Comment:
Would `Math.ceil` work here instead of adding the `0.5` constant? Or does it always need to round up even on "exact" values like `1.0`. Ex: `Math.round(1.0 + .5) = 2.0`. Is that desired?

Reply:
tbh I just used the same logic as the platform for consistency